foxai/visualizer.py

```diff
@@ -191,3 +194,135 @@ def single_channel_visualization(
         figsize=figsize,
         alpha=alpha,
     )
+
+
+def preprocess_object_detection_image(input_image: torch.Tensor) -> np.ndarray:
+    """Process input image to display.
+
+    Args:
+        input_image: Original image.
+
+    Returns:
+        Converted image as np.ndarray in (C x H x W).
+    """
+    return (
+        input_image.squeeze(0)
+        .mul(255)
+        .add_(0.5)
```

Comment:
why do we add 0.5. If initial values are between -0.5:+0.5 shouldn't we do addition first and multiplication second?

Reply:
it is a valid point @adamwawrzynski

Reply:
Yup, it's unnecessary here. I've removed it in this commit: https://github.com/softwaremill/FoXAI/pull/75/commits/bc0b2788b86c2db92262a1632aa07883514f2151.